server/organization/views.py

```diff
@@ -42,23 +44,51 @@ class ModifyOrganizationInventoryItemsDataUpdate(generics.GenericAPIView):
     the Inventory Data is refreshed
     """
 
+    default_refresh_time = 15
+
     # Note: if other methods are added here, keep in mind that the permissions will need to change
     def get_permissions(self):
         permission_classes = PermissionFactory(self.request).get_general_permissions([])
         return [permission() for permission in permission_classes]
 
     def post(self, request):
         data = request.data
-        org_id = data.get('organization', '')
-        new_job_timing = int(data.get('new_job_timing', ''))
+
+        org_id = data.get('organization_id')
+        new_job_timing = int(data.get('time'))
 
         try:
             organization = Organization.objects.get(org_id=org_id)
+            if not new_job_timing:
+                new_job_timing = self.default_refresh_time
             organization.inventory_items_refresh_job = new_job_timing
             organization.save()
-            start_new_job(str(org_id), new_job_timing)
+            start_new_job(org_id, new_job_timing)
             return Response({'detail': 'Time has been updated'}, status=status.HTTP_200_OK)
 
         except Organization.DoesNotExist:
             return Response({'detail': 'Invalid organization'},
                             status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+class ModifyOrganizationInventoryItemFile(generics.GenericAPIView):
+
+    # Note: if other methods are added here, keep in mind that the permissions will need to change
+    def get_permissions(self):
+        permission_classes = PermissionFactory(self.request).get_general_permissions([])
+        return [permission() for permission in permission_classes]
+
+    def post(self, request):
+        data = request.data
+        org_id = data.get('organization_id')
+        file = data.get('file')
+
+        try:
+            organization = Organization.objects.get(org_id=org_id)
+            organization.file = file
+            organization.save()
```

Comment:
Don't we want to write this to a file locally as we discussed?

Reply:
Yes, the files are stored locally in a folder called /media, saving the file here saves the reference name of the file for the organization